packages/labs/task/src/task.ts

```diff
@@ -188,9 +188,6 @@ export class Task<T extends [...unknown[]] = any, R = any> {
         return renderer.complete?.(this.value!);
       case TaskStatus.ERROR:
         return renderer.error?.(this.error);
-      default:
```

Comment:
Let's not remove this

Reply:
It didn't typecheck for me with that part (and also threw errors with my change when I removed a case) but maybe you can help me fix that if that would be the case now again.

On that note `npm run lint` and `npm run format` throws 100s of errors for me. Does the code need to be unstaged or so?